plugins/docsettingtweak.koplugin/directory_defaults.lua

```diff
@@ -0,0 +1,38 @@
+return {--do NOT change this line
+
+ --HELP:
+ -- You can override default settings for documents per directory in this file.
```

Comment:
To prevent people from editing *this* file, maybe the file should be called something like directory_defaults_template.lua instead and not directory_defaults.lua until it's copied over to ./settings?